pkg/util/cmd_utils.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2017 Google, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"os"
+)
+
+var quiet = false
+
+func PrintToStdErr(output string, vars ...interface{}) {
```

Comment:
A helper to add the flag directly in here might be nice, something like:

`func AddFlags(cmd *cobra.Command)`

which could actually do the `cmd.Flags().BoolVar...` stuff. Then users wouldn't need to add the flag or set the constant.